pkg/languagedetection/detector.go

```diff
@@ -6,18 +6,21 @@
 package languagedetection
 
 import (
+	"regexp"
 	"strings"
 
 	"github.com/DataDog/datadog-agent/pkg/languagedetection/languagemodels"
 	"github.com/DataDog/datadog-agent/pkg/process/procutil"
 )
 
+var rubyPattern = regexp.MustCompile(`^ruby\d+\.\d+$`)
```

Comment:
Do we want to do `^ruby(\d+\.\d+)?$` instead? That way if the version is missing we will still match.

Current Version: https://regex101.com/r/pTfnp2/1
Updated: https://regex101.com/r/eXkcQr/1

Reply:
If the version is not present, it will be captured by the "exact match" flow.

Reply:
Ah makes sense 👍🏻